src/main/java/org/broadinstitute/hellbender/tools/exome/FilterTargets.java

```diff
@@ -317,9 +330,159 @@ public String reasonToFilter(final Target target) {
     }
 
     /**
+     * Class that represent a interval of targets to exclude.
+     *
+     * It contains a copy of the interval specification (e.g. "chr1", "chr1:1+", "chr2:100,000,000-200,000,000").
+     *
+     * And the location where it was specified so that we can provide more informative error and target rejection messages.
+     */
+    private final class ExcludeInterval implements Locatable {
```

Comment:
If I understand correctly, this could be a `SimpleInterval` but you want to store e.g. what file each interval came from for error/output.  Personally, I don't believe that this is worth it.  It complicates the code and the most common use case will be a single exclude file, in which case finding the source is trivial.  If an advanced user wants to exclude several interval files I would rather keep the code simple and let the user grep his targets if he needs to troubleshoot.  What are your thoughts on this?